package.json

```diff
@@ -90,4 +90,4 @@
     "vite": "^5.3.4",
     "testcontainers/**/nan": "^2.19.0"
   }
-}
+}
```

Comment:
you have a formatter that removes new lines?

Reply:
had issues in the blst release as well... format on save was doing odd things... hmmm looking into it now

Reply:
ok, fixed